### PR TITLE
Applied code formatting & cleanup [2]

### PR DESCRIPTION
**Pull request by alxsap**
Formatting all code to have a clean state to continue.

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java [hunk 1]
```diff
@@ -67,11 +62,13 @@ static void parse(byte[] b, int len, Map<String, Object> data, boolean saveAnnot
...
             }
         }
-        
+
         String line = new String(b, 0, eol, StandardCharsets.UTF_8);
-        
-        // Would be more efficient to do searches at the byte[] level, but too much bother for now,
-        // especially since there is no standard library method to do offset searches like String has.
+
+        // Would be more efficient to do searches at the byte[] level, but too much
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java]: why? does not exceed line width

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchConsoleLogDecorator.java [hunk 2]
```diff
@@ -18,8 +18,8 @@ public OutputStream decorateLogger(Run run, OutputStream logger) throws IOExcept
         if (config != null) {
             ElasticSearchSender sender = new ElasticSearchSender(null, config);
             return sender.getWrappedLogger(logger);
-        }
-        else return logger;
+        } else
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchConsoleLogDecorator.java]: braces

File: src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchGraphListener.java [hunk 3]
```diff
@@ -23,212 +23,182 @@
...
-        {
-          sendAtomNodeEnd(parent, node);
-        }
-        else if (parent instanceof BlockEndNode)
-        {
-          sendNodeEnd((BlockEndNode<?>)parent);
+    @Override
+    public void onNewHead(FlowNode node) {
+        try {
+            // We cannot send StepEndNodes directly since information is missing, like an
```
Comment (alxsap) [inline src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchGraphListener.java]: comments wrapped

File: src/test/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchTest.java [hunk 4]
```diff
@@ -59,10 +55,7 @@ private void assertLogLine(int lineIndex, String eventType, String message) {
     @Test
     public void testSimplePipelineOutput() throws Exception {
         WorkflowJob project = jenkinsRule.createProject(WorkflowJob.class);
-        project.setDefinition(new CpsFlowDefinition("" +
-                "node {" +
-                "  echo message: 'hello'" +
-                "}", true));
+        project.setDefinition(new CpsFlowDefinition("" + "node {" + "  echo message: 'hello'" + "}", true));
```
Comment (alxsap) [inline src/test/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchTest.java]: keep line breaks

**Comment by alxsap** [comment]
> My IDE still removes some white spaces before the first char in each line
3 spaces instead of 4? 🤔 Any examples?

> I can tell my IDE to only reformat things that have been changed anyways.
👍 

I would merge the PR if there are no bigger blockers anymore.

**Comment by MichaelSp** [comment]
> 3 spaces instead of 4? 

No, just this (occasionally):
image

Yes, please go ahead. No objection from my side.

**Pull request merged**